packages/@lightningjs/ui-components/src/components/Column/Column.test.js

```diff
@@ -399,138 +431,137 @@ describe('Column', () => {
         column.plinko = true;
       });
 
-      // TODO: test is failing because prevItem.core.getAbsoluteCoords in
-      // FocusManager's getIndexOfItemNear is returning NaN. Needs investigating
-      it.skip('should set selected item for item based on previous item', () => {
+      it('should set selected item for item based on previous item', () => {
         const item = column.items[0];
         item.selectedIndex = 3;
         testRenderer.update();
         testRenderer.keyPress('Down');
-        testRenderer.update();
         expect(column.items[1].selectedIndex).toBe(3);
       });
 
-      // TODO: test is failing because prevItem.core.getAbsoluteCoords in
-      // FocusManager's getIndexOfItemNear is returning NaN. Needs investigating
-      it.skip('should select last item in selected row if it is closest', () => {
-        const row = column.items[0];
-        row.items = [...items, { ...baseItem }];
-        row.selectedIndex = row.items.length - 1;
-        testRenderer.update();
-        testRenderer.keyPress('Down');
-        testRenderer.update();
-        expect(column.items[1].selectedIndex).toBe(4);
-      });
-
       it('should select first item if there is only one', () => {
         const row = column.items[1];
         row.items = [{ ...baseItem }];
         testRenderer.update();
         testRenderer.keyPress('Down');
-        testRenderer.update();
-
         expect(row.selectedIndex).toBe(0);
       });
     });
 
     describe('with column height > items', () => {
+      beforeEach(async () => {
+        [column, testRenderer] = createColumn(
+          { items: items.concat(items) },
+          { spyOnMethods: ['_update'] }
+        );
+        await column.__updateSpyPromise;
+        expect(column._Items.y).toBe(0);
+      });
+
       it('should not scroll', () => {
         const [item] = column.items;
         testRenderer.keyPress('Down');
         testRenderer.keyPress('Down');
         testRenderer.keyPress('Down');
-        testRenderer.update();
         expect(item.y).toBe(0);
       });
-
-      // This appears to be a flaky test
-      it.skip('should add items on lazyUpCount', () => {
+      // this is passing but don't believe the test is accurate
```

Comment:
This test is passing but not sure that it's an accurate test

Reply:
Agreed, this test isn't actually testing `lazyUpCount` behavior can probably be removed. The 2 assertions are just expecting that `column.lazyUpCount` was set (2 lines previous in the test) and that an array of 10 items were set to `column.items` (happens in the `beforeEach` of this `describe` block). The calls to `testRenderer.keyPress('Down')` isn't affecting either of those properties.
If it isn't already, `lazyUpCount` behavior should be tested in `NavigationManager`'s tests because that's where that logic lives (separate effort for this though).